ci/tasks/bump-yq-packages/task.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+NEED_COMMIT=false
+
+set -euo pipefail
+
+echo "${PRIVATE_YML}" > postgres-release/config/private.yml
+
+pushd postgres-release
+  CURRENT_BLOBS=$(bosh blobs)
+  CURRENT_VERSION=$(curl -s -L https://api.github.com/repos/mikefarah/yq/releases/latest | grep "tag_name" | sed s/\"tag_name\":\//g | sed s/\"//g | sed s/\,//g | sed s/v//g | xargs)
+  mv ../yq-src/yq_darwin_amd64 ../yq-src/yq-${CURRENT_VERSION}
```

Comment:
I'm guessing the `darwin` here was left over from testing and should be changed to linux

Reply:
Done